LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017 The KubeVirt Authors
+   This file is part of the KubeVirt project
```

Comment:
Do we want to change the `LICENSE` file? @enp0s3

Reply:
we have to change it to fit the right text no?

Reply:
@fossedihelm +1, from what I see in other projects, its uncommon to modify the license file.
@avlitman Lets drop this modification and stick with the hack/boilerplate as the default template to use.

Reply:
done